frontend/controllers/UniversitiesController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace frontend\controllers;
+
+use Yii;
+use yii\web\Controller;
+use frontend\models\University;
+use frontend\models\Department;
+use frontend\models\Discipline;
+use frontend\models\Students;
+
+class UniversitiesController extends Controller
+{
+    public function actionIndex()
+    {
+        return $this->render('index');
+    }
+
+    public function actionUniversity()
+    {
+        $model = new University();
+        $dataProvider = $model->search(Yii::$app->request->queryParams);
+
+        return $this->render('indexuniversity', [
+            'searchModel' => $model,
+            'dataProvider' => $dataProvider,
+        ]);
+    }
+
+    public function actionCreateuniversity()
+    {
+        $model = new University();
+
+        if (count(Yii::$app->request->post('University'))) {
+            $university = Yii::$app->request->post('University');
+            $model->name = $university['name'];
```

Comment:
you can use $model->load(Yii::$app->request->post('University')) with your validation rules should set all the fields